descriptors/ModuleDescriptor-template.json

```diff
@@ -2303,6 +2305,8 @@
       "description" : "Internal permission set for fetching item(s)",
       "subPermissions": [
         "inventory-storage.items.item.get",
+        "circulation-item-storage.items.item.get",
```

Comment:
@MagzhanArtykov The permissions which added here is part of mod-circulation-item module. If I am not wrong, mod-circulation-item will not be enabled in all environments. Will it create any problem when the module is not enabled but the permission is added here.

Reply:
Tested. Seems okapi ignores these permissions